translator.softHSM.Dockerfile

```diff
@@ -0,0 +1,61 @@
+#
+#   Docker Image for Translator Component in Test environments only.
+#   Translator depends on softHSM and OpenSC installations in Test environments.
+#
+FROM gradle:jdk11 AS build
+
+WORKDIR /app
+USER root
+ENV GRADLE_USER_HOME ~/.gradle
+
+COPY build.gradle build.gradle
+COPY settings.gradle settings.gradle
+COPY proxy-node-shared/ proxy-node-shared/
+
+ARG component=proxy-node-gateway
+COPY ${component}/src ${component}/src
+COPY ${component}/build.gradle ${component}/build.gradle
+
+RUN gradle --no-daemon --quiet -p ${component} --parallel install -x test
+ENTRYPOINT ["gradle", "--no-daemon"]
```

Comment:
From this line up, and from line 58 down, the Dockerfiles are the same as the base Dockerfile used to build the other images. Instead of the duplication, could you extend that Dockerfile with the extra bits here?